cmd/finch/virtual_machine_init.go

```diff
@@ -94,11 +95,17 @@ func (iva *initVMAction) run() error {
 	instanceName := fmt.Sprintf("--name=%v", limaInstanceName)
 	limaCmd := iva.creator.CreateWithoutStdio("start", instanceName, iva.baseYamlFilePath, "--tty=false")
 	iva.logger.Info("Initializing and starting Finch virtual machine...")
+	// Show spinner
+	s := spinner.New(spinner.CharSets[11], 100*time.Millisecond)
+	s.Suffix = "VM creation in progress"
+	s.Start()
 	logs, err := limaCmd.CombinedOutput()
 	if err != nil {
+		s.Stop()
 		iva.logger.Errorf("Finch virtual machine failed to start, debug logs: %s", logs)
 		return err
 	}
+	s.Stop()
```

Comment:
You've got this twice - consider using a `defer s.Stop()` instead after `s.Start()`

Reply:
Thanks, updated the code.

Reply:
Currently the logs are printed _before_ the spinner stops, while we may want them to be printed after instead.

```go
	if err != nil {
		iva.logger.Errorf("Finch virtual machine failed to start, debug logs: %s", logs)
		return err
	}
	iva.logger.Info("Finch virtual machine started successfully")
```

As a result, could you wrap the following statements in a func (`initVM`?) print the logs above after calling that func? Thanks!

```go
	instanceName := fmt.Sprintf("--name=%v", limaInstanceName)
	limaCmd := iva.creator.CreateWithoutStdio("start", instanceName, iva.baseYamlFilePath, "--tty=false")
	iva.logger.Info("Initializing and starting Finch virtual machine...")
	// Show spinner
	s := spinner.New(spinner.CharSets[11], 100*time.Millisecond)
	s.Suffix = "VM creation in progress"
	s.Start()
	defer s.Stop()
	logs, err := limaCmd.CombinedOutput()
```